packages/code/README.md

```diff
@@ -123,3 +123,158 @@ interface LanguageOption {
+  const testUtils2 = getTestUtils('lg-code-2'); // data-lgid
+
+  // First Code
+  expect(testUtils1.getLanguage()).toBe('javascript');
+
+  // Second Code
+  expect(testUtils2.getLanguage()).toBe('python');
+});
+```
+
+### Test Utils
+
+```tsx
+const {
+  getLanguage,
+  getTitle,
+  getLanguageSwitcher: {
+    getInput,
+    getAllOptions,
+    getOptionByValue,
+    isDisabled,
+  },
+  getIsLoading,
+  getCopyButton: { getButton, isDisabled },
+  getExpandButton: { getButton, isExpanded },
+} = getTestUtils();
+```
+
+| Util                    | Description                                              | Returns                 |
+| ----------------------- | -------------------------------------------------------- | ----------------------- |
+| `getLanguage()`         | Returns the current language of the code block           | `string`                |
+| `getTitle()`            | Returns the title of the code block                      | `string` \| `null`      |
+| `getLanguageSwitcher()` | Returns utils for interacting with the language switcher | `LanguageSwitcherUtils` |
+| `getIsLoading()`        | Returns whether the code block is in loading state       | `boolean`               |
+| `getCopyButton()`       | Returns utils for interacting with the copy button       | `CopyButtonUtils`       |
+| `getExpandButton()`     | Returns utils for interacting with the expand button     | `ExpandButtonUtils`     |
+
+### LanguageSwitcherUtils
+
+| Util                              | Description                                        | Returns                 |
+| --------------------------------- | -------------------------------------------------- | ----------------------- |
+| `getInput()`                      | Returns the input element of the language switcher | `HTMLElement` \| `null` |
+| `getAllOptions()`                 | Returns all options in the language switcher       | `Array<HTMLElement>`    |
+| `getOptionByValue(value: string)` | Returns the option element by its value            | `HTMLElement` \| `null` |
+| `isDisabled()`                    | Returns whether the language switcher is disabled  | `boolean`               |
+
+### CopyButtonUtils
+
+| Util           | Description                                 | Returns                 |
+| -------------- | ------------------------------------------- | ----------------------- |
+| `getButton()`  | Returns the copy button element             | `HTMLElement` \| `null` |
+| `isDisabled()` | Returns whether the copy button is disabled | `boolean`               |
+
+### ExpandButtonUtils
+
+| Util           | Description                                | Returns                 |
+| -------------- | ------------------------------------------ | ----------------------- |
+| `getButton()`  | Returns the expand button element          | `HTMLElement` \| `null` |
+| `isExpanded()` | Returns whether the code block is expanded | `boolean`               |
```

Comment:
I'm not sure if including `isExpanded` here makes sense. The util is `getExpandButton`, but `isExpanded` doesn't really need to be associated with the button since it's testing whether or not the code snippet is expanded or collapsed.